.github/workflows/run-acceptance-tests.yml

```diff
@@ -105,6 +105,70 @@ jobs:
         parallel: [3]
         index: [0, 1, 2]
 
+  integration-test:
```

Comment:
This is duplicated right? How difficult would it be to factor it out into a shared code.

In AWS we have:


```
    - name: Setup tools
      uses: ./.github/actions/setup-tools
      with:
        tools: go, pulumictl, pulumicli, schema-tools
```

AS an example of a form permitting sharing.